src/FilesSidebarTabApp.vue

```diff
@@ -19,26 +19,294 @@
   - along with this program. If not, see <http://www.gnu.org/licenses/>.
 -->
 <template>
-	<p>Talk tab coming soon</p>
+	<div class="talkChatTab">
+		<div v-if="fileInfo && fileInfo.isDirectory()" class="emptycontent">
+			<div class="icon icon-talk" />
+			<h2>Conversations are not available for folders</h2>
+		</div>
+		<div v-else-if="isTalkSidebarSupportedForFile === undefined" class="emptycontent ui-not-ready-placeholder">
+			<div class="icon icon-loading" />
+		</div>
+		<div v-else-if="!isTalkSidebarSupportedForFile" class="emptycontent file-not-shared">
+			<div class="icon icon-talk" />
+			<h2>{{ t('spreed', 'Discuss this file') }}</h2>
+			<p>{{ t('spreed', 'Share this file with others to discuss it') }}</p>
+			<button class="primary" @click="openSharingTab">
+				{{ t('spreed', 'Share this file') }}
+			</button>
+		</div>
+		<div v-else-if="isTalkSidebarSupportedForFile && !token" class="emptycontent room-not-joined">
+			<div class="icon icon-talk" />
+			<h2>{{ t('spreed', 'Discuss this file') }}</h2>
+			<button class="primary" @click="joinConversation">
+				{{ t('spreed', 'Join conversation') }}
+			</button>
+		</div>
+		<template v-else>
+			<button class="call-button primary" :disabled="true">
+				Calls will return soon
+			</button>
+			<ChatView :token="token" />
+		</template>
+	</div>
 </template>
 
 <script>
+// Explicitly enable smart-tabs so it is also applied to the CSS rules.
+/* eslint no-mixed-spaces-and-tabs: ["error", "smart-tabs"] */
```

Comment:
```suggestion
```

No hiding please, the coding style is the same across all nextcloud vue apps

Reply:
It is not hiding anything, just enabling smart tabs so the comments can use spaces after the tabs for proper alignment (it works by default in the JavaScript code, but I have no idea why it does not in the CSS code). That is, to be able to write
```
	/* The chat view shares its parent with the call button, so the default
	 * "height: 100%" needs to be unset. */
```
instead of
```
	/* The chat view shares its parent with the call button, so the default
	* "height: 100%" needs to be unset. */
```

But as I said I would expect that to work too for the CSS code by default, so I do not know what I am missing.

Reply:
Well then this should be added to https://github.com/nextcloud/eslint-config-nextcloud/blob/master/index.js